src/components/chakra-next-image.tsx

```diff
@@ -1,7 +1,7 @@
-import { chakra } from '@chakra-ui/system'
 import { Flex } from '@chakra-ui/layout'
 import type { FlexProps } from '@chakra-ui/layout'
-import NextImage, { ImageProps, ImageLoaderProps } from 'next/image'
+import { chakra } from '@chakra-ui/system'
+import NextImage, { ImageLoaderProps, ImageProps } from 'next/image'
```

Comment:
Let's add `type ImageLoaderProps, type ImageProps` to the props imports. We can also do that to the `FlexProps` above and merge the first two lines together.

Reply:
Right good catch will do later :)

Reply:
I tried out the inline syntax, but it wouldn't let the application build. Maybe we need a newer next version.